controllers/rcj_soccer_referee_supervisor/referee/supervisor.py

```diff
@@ -213,3 +222,11 @@ def reset_robot_position(self, robot):
 
         # Ensure the progress checker does not count this "jump"
         self.progress_chck[robot].reset()
+
+    def robot_to_team_name(self, robot) -> str:
```

Comment:
Forgot to type robot.
```suggestion
    def robot_to_team_name(self, robot: str) -> str:
```